object.go

```
@@ -469,11 +469,7 @@ func (d *Decoder) decInstance(typ reflect.Type, cls *classInfo) (interface{}, er
 			if err != nil {
 				return nil, perrors.Wrapf(err, "decInstance->Decode field name:%s", fieldName)
 			}
-			v, ok := b.(bool)
```

@xfstart07  why ignore the error？

---

我看 java 的 Boolean 类型实际数据有 N F T 这类,如果是 N 的话,实际 b 是个 nil ,在 go 中是给默认值 false

---

currently not support java Boolean, there should not ignore the error, use perrors.Errorf instead pls.